<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmwCellAnalyzer</class>
 <widget class="QMainWindow" name="QmwCellAnalyzer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QmwCellAnalyzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <widget class="QWidget" name="tabApp">
       <attribute name="title">
        <string>App</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>301</width>
          <height>401</height>
         </rect>
        </property>
        <property name="title">
         <string>Experiment Setting</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Type</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="comboExpriType"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Person</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="editPerson"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="editExpriName"/>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Sample ID</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="editSampleId"/>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Cell Type</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QComboBox" name="comboCellType"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Cell Size</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QComboBox" name="comboSize"/>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Dilution Ratio</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QComboBox" name="comboDilutionRatio"/>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Data/Time:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QLineEdit" name="editTime"/>
         </item>
         <item row="8" column="1" colspan="2">
          <widget class="QPushButton" name="btnConfirmInfo">
           <property name="text">
            <string>Confirm</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>319</x>
          <y>9</y>
          <width>251</width>
          <height>401</height>
         </rect>
        </property>
        <property name="title">
         <string>Chamber Select</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lblChamber">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>357</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="res.qrc">:/images/samle_plate.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="2,2,2,2,2,1">
           <property name="spacing">
            <number>36</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="topMargin">
            <number>30</number>
           </property>
           <item>
            <widget class="QCheckBox" name="cBoxChamber1">
             <property name="text">
              <string>Chamber1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cBoxChamber2">
             <property name="text">
              <string>Chamber2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cBoxChamber3">
             <property name="text">
              <string>Chamber3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cBoxChamber4">
             <property name="text">
              <string> Chamber4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cBoxChamber5">
             <property name="text">
              <string>Chamber5</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>416</y>
          <width>557</width>
          <height>27</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,1">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>121</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnStart">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>342</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
